Directory.Build.targets

```diff
@@ -8,7 +8,7 @@
     <PackageReference Update="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
     <PackageReference Update="Microsoft.Extensions.FileSystemGlobbing" Version="2.0.1" />
     <PackageReference Update="Microsoft.NET.Test.Sdk" Version="16.5.0" />
-    <PackageReference Update="Microsoft.TestPlatform.ObjectModel" Version="16.5.0" />
+    <PackageReference Update="Microsoft.TestPlatform.ObjectModel" Version="16.9.1" />
```

Comment:
can you add a link above with issue ref?

Reply:
Done